Bottom: 7a7197b0e0adecfc1646d108f8d5d1ea31ca34b3
Top:    ff1d3f1895b0f5b7073a5c31cc0d287c1e0c051a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-26 09:12:42 -0500

Refresh of all-docker_container

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 0479f7c7..cb19329d 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -141,9 +141,9 @@
       - '{{ openssl_dv_name }}'
       - '{{ wiki_dv_name }}'
     env:
-      - WIKI_HOSTNAME: '{{ container_addr_map.wiki.hostname }}'
-      - WIKI_MAIL_USER: 'wiki'
-      - WIKI_MAIL_PASSWORD: '{{ wiki_email_password | quote }}'
+      WIKI_HOSTNAME: '{{ container_addr_map.wiki.hostname }}'
+      WIKI_MAIL_USER: 'wiki'
+      WIKI_MAIL_PASSWORD: '{{ wiki_email_password | quote }}'
     image: '{{ wiki_image_name }}:{{ docker_image_tag }}'
     state: started
